proto/viam/app/v1/app.proto

```diff
@@ -843,11 +859,13 @@ enum FragmentErrorType {
   FRAGMENT_ERROR_TYPE_CYCLE_DETECTED = 4;
 }
 
+// Cached fragment usage statistics
 message FragmentUsage {
   string fragment_id = 1;
   int32 organizations = 2;
   int32 machines = 3;
   int32 machines_in_current_org = 4;
+  string version = 5;
```

Comment:
optional?

Reply:
I like `optional` for intent a little more

Reply:
+1 to optional.